lib/tasks/message_queue.rake

```diff
@@ -6,6 +6,16 @@
 require 'rummager'
 
 namespace :message_queue do
+  desc "Create queues"
+  task :create_queues do
+    bunny = Bunny.new(ENV["RABBITMQ_URL"])
+    channel = bunny.start.create_channel
+    exch = Bunny::Exchange.new(channel, :topic, "published_documents")
+    channel.queue("rummager_to_be_indexed").bind(exch, routing_key: "*.links")
+    channel.queue("rummager_bulk_reindex").bind(exch, routing_key: "*.bulk.reindex")
+    channel.queue("rummager_govuk_index").bind(exch, routing_key: "*.*")
```

Comment:
Do we need to do any binding here?

I wonder if we should be using [#queue_declare](http://www.rubydoc.info/github/ruby-amqp/bunny/Bunny%2FChannel%3Aqueue_declare) ?

Reply:
The queue will need to be bound to the exchange otherwise messages from the exchange won't be routed into the queue.  Are you proposing the binding gets done elsewhere?

We could switch over to ` queue_declare `, but ` queue ` command will have the benefit of being idempotent.

Reply:
Ah sure, I thought the govuk_message_queue_consumer did the bindings but I see that I'm wrong. Don't mind me 😄